docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3"
+services:
+  postgres:
+    image: postgres
+    environment:
+      - POSTGRES_USER=projectmatch
+      - POSTGRES_PASSWORD=projectmatchpass
+    volumes:
+      - ./db/postgres:/var/lib/postgresql/data
+  rails:
+    build: 
+      context: .
+      dockerfile: docker/Dockerfile
+    depends_on:
+      - postgres
+    links:
+      - postgres
+    volumes:
+      - ./:/usr/src/app
+    ports:
+      - "3000:3000"
```

Comment:
Silly EOL error that might return to haunt us later. 

Consider using a Ruby formatter in a `.gitattribute` file that will automatically handle this, along with other formatting issues, when the changes are pushed to our remote. 

For Python, I use a PEP-8 formatter called Black; I found [this for Ruby](https://github.com/ruby-formatter/rufo), maybe this'll be helpful in our case.